src/Core/Checkout/Cart/Delivery/DeliveryCalculator.php

```diff
@@ -122,7 +122,11 @@ private function calculateDelivery(CartDataCollection $data, Cart $cart, Deliver
 
         if (!$costs) {
             $cart->addErrors(
-                new ShippingMethodBlockedError((string) $shippingMethod->getTranslation('name'))
```

Comment:
maybe the technical name would make more sense? it is also unique, but more readable. 
also this would not cause a break in the error creation :thinking: